mozdownload/scraper.py

```diff
@@ -816,6 +816,28 @@ def detect_platform(self):
 
         return platform
 
+    def is_build_dir(self, dir):
+        """Return whether or not the given dir contains a build."""
+
+        url = urljoin(self.base_url, self.build_list_regex, dir)
+
+        if self.application in MULTI_LOCALE_APPLICATIONS \
+                and self.locale != 'multi':
+            url = urljoin(url, self.locale)
```

Comment:
To sum it up for all. We left this part in, as it only affects b2g builds at the moment, which we do not officially support. In case removing this part leads to a regression, we leave it for now.